docs/intro/dev.md

```diff
@@ -145,8 +145,7 @@ Transactions fees are calculated based on two main parts:
 The more work a transaction requires, the more compute units it will use, and
 the more it will cost.
 
-You can read more about transaction fees
-[here](/docs/core/transactions/fees.md).
+You can read more about transaction fees [here](/docs/intro/transaction_fees).
```

Comment:
this link is invalid. it should be `/docs/intro/transaction-fees`

Reply:
change to `/docs/core/fees` for the updated fee page